golem-rpc-api/src/net.rs

```diff
@@ -141,6 +142,13 @@ pub struct AclRuleItem<Identity> {
     pub deadline: Option<chrono::DateTime<chrono::Utc>>,
 }
 
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ACLResult {
+    pub success: bool,
+    pub exist: Option<Vec<String>>,
```

Comment:
```suggestion
    pub exist: Vec<String>,
```
It just might be empty

Reply:
nice one 👍